src/attestation/azext_attestation/tests/latest/test_attestation_scenario.py

```diff
@@ -100,8 +93,7 @@ def call_scenario(test, rg):
     setup(test, rg)
     step_attestationproviders_create(test, rg)
     step_attestationproviders_get(test, rg)
-    step_attestationproviders_list(test, rg)
```

Comment:
No need to test "list" operation?

Reply:
> No need to test "list" operation?

Added `list` again. Please see the latest code.